src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -22,6 +22,7 @@
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('annotations.reader', AnnotationReader::class)
+            // @deprecated since Symfony 6.3, to be removed in 7.0
```

Comment:
This should not even be removed in 7.0. The deprecation warning is not triggered when using both the attribute and the annotation. So when migrating to 7.0, we won't be in a case where you have a code base without `@required` (your codebase won't rely on it being effective but it can still have it to support multiple versions of Symfony). And so we cannot stop telling the Doctrine annotation reader to ignore it.